src/imgui/ImGuiMachine.cc

```diff
@@ -201,13 +201,17 @@ void ImGuiMachine::showMenu(MSXMotherBoard* motherBoard)
 				};
 				if (!saveSetupOpen) {
 					// on each re-open of this menu, create a suggestion for a name
-					saveSetupName = motherBoard->getMachineName();
+					auto configName = motherBoard->getMachineName();
+					auto* info = findMachineInfo(configName);
+					auto initialSaveSetupName = info ? info->displayName : configName;
+					saveSetupName = initialSaveSetupName;
 					if (exists()) {
 						saveSetupName = FileOperations::stem(FileOperations::getNextNumberedFileName(
-							Reactor::SETUP_DIR, motherBoard->getMachineName(), Reactor::SETUP_EXTENSION, true));
+							Reactor::SETUP_DIR, initialSaveSetupName, Reactor::SETUP_EXTENSION, true));
 					}
 				}
 				ImGui::InputText("##save-setup-name", &saveSetupName);
+				simpleToolTip(saveSetupName);
```

Comment:
Why this? Left over from some experiment?
A tooltip showing the exact same string as the input string doesn't seem useful.

Reply:
It is useful: the name can be longer than the input box has room for. With the tooltip you can easily view the full name. At least, for this reason I added it.